build.gradle

```diff
@@ -104,7 +104,7 @@ project.ext.externalDependency = [
   'spock': 'org.spockframework:spock-core:1.3-groovy-2.5',
   'testng': 'org.testng:testng:6.13.1',
   'velocity': 'org.apache.velocity:velocity-engine-core:2.2',
-  'zero_allocation_hashing': 'net.openhft:zero-allocation-hashing:0.7',
+  'zero_allocation_hashing': 'net.openhft:zero-allocation-hashing:0.16',
```

Comment:
this can use the custom fork now, right?

Reply:
Not yet; the custom fork is only released internally to LinkedIn. We need to update internal users to use our fork, and then change their code to use `xx()` instead of `xx_r39()`. After that, we are free to use `net.openhft:zero-allocation-hashing:0.16` in Pegasus. We won't need to fork anymore at that point.

Reply:
got it. Thanks!